### ISSUE TITLE
[CoE Starter Kit - QUESTION] Compliance Parameters for flows

### ISSUE THREAD

**Issue by PPMini** [microsoft/coe-starter-kit#8342]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi Team, 

Can we have compliance parameters for flows as well? Is it the future roadmap?

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

April 2024

### What app or flow are you having the issue with?

Governance - Solution

### What method are you using to get inventory and telemetry?

Cloud flows

AB#3351

**Comment by Jenefer-Monroe** [comment]
What types of things would you see being the trigger here? We do have something on the backlog but its not entirely clear what would signify a flow that needs compliance information, which is why its manual today.

**Comment by PPMini** [comment]
Number of users shared /group shared?

**Comment by Jenefer-Monroe** [comment]
gotcha. We dont have that information stored yet, but we will be working on it soon as we are no longer blocked. 
I can place this on the backlog for potential funding once that is in place.

**Comment by PPMini** [comment]
Perfect! Thanks.

**Comment by AmarSaiRam** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: FEATURE TITLE Consolidated Compliance Enhancements: Unified Improvemen…](https://github.com/microsoft/coe-starter-kit/issues/10324)

Thank you for your contributions
